azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -193,18 +196,22 @@ public AzkabanWebServer() throws Exception {
    * Constructor
    */
   public AzkabanWebServer(Server server, Props props) throws Exception {
```

Comment:
should we make the constructor private since this class seems to be implemented as singleton